plugins/modules/repository_set_info.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2021 William Bradford Clark
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: repository_set_info
+version_added: 2.1.0
+short_description: Fetch information about Red Hat Repositories
+description:
+  - Fetch information about Red Hat Repositories
+author: "William Bradford Clark (@wbclark)"
+options:
+  name:
+    description:
+      - Name of the repository set
+    required: false
+    type: str
```

Comment:
that's part of the base `infomodule` doc, so you don't *need* to specify it (unless you want to override something):

https://github.com/theforeman/foreman-ansible-modules/blob/292dc967ee203660c4628cfcf3edf12fc0278590/plugins/doc_fragments/foreman.py#L331-L336